code/client/kstr.c

```diff
@@ -58,7 +58,7 @@ struct kstr kstr_substr(const struct kstr *str, size_t pos, size_t length)
     return (struct kstr){ .length = length, .capacity = capacity, .buffer = str->buffer + pos };
 }
 
-bool kstr_read(struct kstr *str, const uint16_t *src, size_t size)
+bool kstr_read(struct kstr *str, uint16_t *src, size_t size)

```

Comment:
The `const` doesn't seem like it needed to be removed. Is there any issue?

Reply:
The reason is in the commit message but it’s being assigned to kstr buffer which isn’t a const

Reply:
Right, I think I solve that this morning when looking at the static assert issue.

Maybe just check if it works without this change, I think it will.